Sources/Settings/Controller/SettingsController.swift

```diff
@@ -110,14 +108,6 @@ class SettingsController: UITableViewController {
         navigationItem.rightBarButtonItem = tipJarBarButton
     }
 
-    private func setNavBarAppearance() {
```

Comment:
now navigationBar changes color without this code (cause of extension) - and without lag!